test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -57,6 +65,11 @@
           webserver-config (:webserver default-config)]
       ;; use `-with-config` variant, so that we can pass in the entire config map
       (bootstrap/with-puppetserver-running-with-mock-jrubies
+       "MOCK.TODO: discuss - this is *probably* ok because all we really care about
+        is that the webserver is accepting connections and handing them to the correct
+        ring handler.  Would be better if we could implement this to hit a clojure
+        endpoint instead of a Ruby one but I don't remember if there are any pure-clojure
+        endpoints under /puppet."
```

Comment:
static-file-content might work here, too, although it seems onerous to have to set that up for this test.  As with an earlier comment, I think if we provided a way to echo the request map from `.handleRequest()` back out though the response and could use that to just validate that the request was handed off correctly that I'd be fine with leaving this as a mock and using the "node" endpoint.  If you're opposed to that whole approach, though, maybe it'd be better to convert this back to using a real JRubyPuppet and add some more assertions to see that we got a response body which is appropriate for a "node" request.